runtime/precompiles/src/lib.rs

```diff
@@ -62,6 +93,11 @@ where
 			a if a == hash(6) => Some(Bn128Add::execute(input, target_gas, context)),
 			a if a == hash(7) => Some(Bn128Mul::execute(input, target_gas, context)),
 			a if a == hash(8) => Some(Bn128Pairing::execute(input, target_gas, context)),
+			// Moonbeam precompiles :
+			a if a == hash(255) => Some(Dispatch::<R>::execute(input, target_gas, context)),
+			a if a == hash(256) => Some(ParachainStakingWrapper::<R>::execute(
```

Comment:
Addresses need to change according to the standard Gorka introduced in https://github.com/PureStake/moonbeam/pull/323